src/modules/top-level.nix

```diff
@@ -194,6 +200,21 @@ in
       ln -s ${profile} .devenv/profile
     '';
 
+    packages =
+      let
+        mergeInputs = attrPath:
+          lib.subtractLists config.inputsFrom (builtins.concatMap (lib.attrByPath attrPath [ ]) config.inputsFrom);
+        allInputs = builtins.filter lib.isDerivation (builtins.concatMap mergeInputs [
+          [ "buildInputs" ]
+          [ "nativeBuildInputs" ]
+          [ "propagatedBuildInputs" ]
+          [ "propagatedNativeBuildInputs" ]
+          [ "stdenv" "initialPath" ]
+          [ "stdenv" "defaultBuildInputs" ]
+          [ "stdenv" "defaultNativeBuildInputs" ]
+        ]);
+      in
+      lib.mkAfter allInputs;
```

Comment:
This will most likely have a lot of duplicate packages when passed multiple packages, how is this handled.
Packages added like this should appear in search paths after any explicitly added packages.